sourcedata/README.md

```diff
@@ -0,0 +1 @@
+The folder `sourcedata` contains all raw images obtained from the differents sites. These images are then preprocessed uisng the `proecss_data.sh` script and the preprocessed images are included in the root.
```

Comment:
Which `process_data.sh` script is this? I don't see it in this repository, but maybe you mean [this one](https://github.com/spine-generic/spine-generic/blob/master/process_data.sh)? Or [this other one](https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/blob/v2.2/processing_spine_generic/process_data.sh)? (And which version?)

Reply:
This one, however, I modified it on the branch: https://github.com/spine-generic/spine-generic/pull/281 I would need someone to accept the changes, and create a release after with the newest version

Reply:
We can still link to the file at a specific commit, even without merging anything, so I'll do that:
https://github.com/spine-generic/spine-generic/blob/e01bfe43bf81313b8c8713bb46b14c7bab23eb98/process_data.sh